ideascube/models.py

```diff
@@ -239,7 +239,8 @@ def index_strings(self):
         _('Current occupation'), choices=OCCUPATION_CHOICES, blank=True,
         max_length=32)
     country_of_origin_occupation = models.CharField(
-        _('Occupation in the country of origin'),
+        _('Occupation before reaching {place}').format(
+                                        place=settings.IDEASCUBE_PLACE_NAME),
```

Comment:
So the verbose name is depending on a setting, but it's set by the migration to... the value from your setting at the time you generated the migration.

Is that really how it should work?